README.md

```diff
@@ -1,31 +1,61 @@
-# NF1_SchwannCell_data_analysis
-Here we analyze [NF1](https://www.sciencedirect.com/science/article/pii/B9780444529022000539) morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data) repository.
-Our analysis consists of finding interesting patterns in the NF1 cell morphology data from the [nf1_cellpainting_data](https://github.com/WayScience/NF1_SchwannCell_data_analysis) repository, and then developing, evaluating and interpretting our NF1 model.
-The NF1 model is a binary classifcation model optimized from NF1 cell morphology data to classify cell genotypes {Null, WT} for the NF1 gene.
+# NF1 Schwann cell data analysis
+
+We aim to characterize cell morphology signatures of neurofibromin in Schwann cells. 
+We applied a modified Cell Painting assay on two isogenic Schwann cell lines, one of the wildtype genotype (*NF1+/+*) and one of the null genotype (*NF1-/-*), both from the same patient.
+The modified assay stains for four organelles: nuclei, endoplasmic reticulum, mitochondria, and actin. 
+We applied CellProfiler pipelines to perform quality control, illumination correction, segmentation, and feature extraction. 
+
+![Figure 1](./3.figures/figures/main_figure_1_workflow.png)
+> *Image montage of dataset and analysis workflow.* (A) The workflow of our analysis pipeline demonstrates the steps taken from image analysis to machine learning. (B) Example image montages of the Cell Painting channels and composite image (all channels overlayed) for each NF1 genotype. The scale bar represents 25 μM.
```

Comment:
You are right, fixed it! And in this case I am not sure of a standard, but that scale looks to be around what the average cell size is. And I don't do this in this figure but I do it in another figure where the 25 uM is in the photo. In this case, I think it will work either way, but I don't see where adding the 25 uM will make a big improvement at this time. Thanks!